File: Corpo.Domain/Models/Promotion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Corpo.Domain.Models
{
    public class Promotion
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int DiscountMainMember { get; set; }
        public DateTime From { get; set; }
        public DateTime To { get; set; }
        public virtual List<PromotionAnotherMember> PromotionAnotherMember { get; set; }

    }
}

File: Corpo.Web/ClientApp/src/app/domain/promotion.ts
import { PromotionAnotherMember } from "./promotion-another-member";

export class Promotion {
  id: number;
  name: string;
  from: string;
  to: string;
  discountMainMember: number;
  promotionAnotherMember: PromotionAnotherMember[];
}
